src/dfx/assets/new_project_node_files/webpack.config.js

```diff
@@ -93,6 +94,11 @@ module.exports = {
       Buffer: [require.resolve("buffer/"), "Buffer"],
       process: require.resolve("process/browser"),
     }),
+    new CopyPlugin({
```

Comment:
Why is this needed?  Asset sync copies directly from the `source` directories, so it should see these files without having to copy them.

Reply:
`source` directories in `dfx.json` only include `src/__project_name__/assets` and `src/__project_name__/dist`. Webpack builds stuff from `src/__project_name__/src` into `src/__project_name__/dist`, however since `.ic-assets.json` is not being used in JS files, it's not being moved over from `src` to `dist`, hence we need to manually copy it

Reply:
tho admittedly, that's not the perfect place to bring this change. sorry for that